src/languages/smarty.coffee

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+  name: "Smarty"
```

Comment:
Will need to add a Beautifier to support this language.
For example: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/js-beautify.coffee#L9 and https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L50